mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,101 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    function __construct()
+    {
+
+    }
+
+    public function getCSVHeaderForStatsExp($experiment)
+    {
+        if (strcmp($experiment, "ex1") === 0) {
+            return "detector,project,synthetics,misuses,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        } elseif (strcmp($experiment, "ex2") === 0) {
+            return "detector,project,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        } else {
+            return "detector,project,misuses,potential_hits,open_reviews,need_clarification,yes_agreements,no_agreements,total_agreements,yes_no_agreements,no_yes_agreements,total_disagreements,kappa_p0,kappa_pe,kappa_score,hits,recall";
+        }
+    }
+
+    public function createCSVFromStats($experiment, $stats)
+    {
+        $result = "sep=,\n" . $this->getCSVHeaderForStatsExp($experiment);
+        foreach ($stats as $key => $stat) {
+            $result .="\n";
+            $result .= "{$stat->getDisplayName()},{$stat->number_of_projects},";
```

Comment:
I would actually also prefer collecting the values in an array and imploding rows at the end. In fact, we could event collect the data in a multidimensional array, like in 

```
$csv["column1"] = ["row1val", "row2val", ...];
$csv["column2"] = ["row1val", "row2val", ...];
...
```

and construct the header and rows from this structure in a helper method (which we could reuse for the different CSVs).